packages/core/i18n/README.md

```diff
@@ -395,6 +397,42 @@ Every single method listed in [FormatJS](https://formatjs.io/docs/intl) is expos
 
 (as previously exposed by vue18n-n)
 
+### formatUnixTimeStamp
+
+Formats a unix timestamp into a formatted date string
+
+`code:`
+
+```ts
+const { formatUnixTimeStamp } = useI18n()
+console.log(formatUnixTimeStamp('1558006979'))
+console.log()
```

Comment:
nit: is this empty log needed? There's a couple instances in the docs file.
```suggestion
```

Reply:
Nope not needed, removed 'em all